torch/_functorch/_aot_autograd/runtime_wrappers.py

```diff
@@ -1887,8 +1891,10 @@ def get_types_for_tangents(tangents):
                         )
                         for i, t in enumerate(all_args)
                     ]
-                    all_args = unwrap_tensor_subclasses(
-                        all_args, is_joint_structure=False
+
+                    all_args = runtime_unwrap_tensor_subclasses(
+                        all_args,
+                        append_symints=False,
```

Comment:
Can you add a comment on why `append_symints` is False here?

I think the answer we agreed on is that any symints coming from tensor subclasses should always come from primals, and so they will show up as extra arguments to the forward graph, and they will be saved for backward as activations (instead of needing to be generated as extra tangent arguments in the backward). I think you also initially tried to include symints here, and ran into some issues but I can't remember what they were. Does that sound right? Aand if you remember what broke that would be helpful!